bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/BahmniVisitLocationController.java

```diff
@@ -24,12 +27,41 @@ public BahmniVisitLocationController(BahmniVisitLocationService bahmniVisitLocat
     }
 
 
-    @RequestMapping(method = RequestMethod.GET, value = "/{loginLocationUuid}")
+    @RequestMapping(method = RequestMethod.GET, value = "/visitLocation/{loginLocationUuid}")
     @ResponseBody
     public HashMap<String, String> getVisitLocationInfo(@PathVariable("loginLocationUuid") String locationUuid ) {
         HashMap<String, String> visitLocation = new HashMap<>();
         visitLocation.put("uuid",bahmniVisitLocationService.getVisitLocationUuid(locationUuid));
         return visitLocation;
     }
 
+    @RequestMapping(method = RequestMethod.GET, value = "/facilityLocation/{locationUuid}")
+    @ResponseBody
+    public HashMap<String, LocationSummary> getFacilityVisitLocationInfo(@PathVariable("locationUuid") String locationUuid ) {
```

Comment:
why do we need Hashmap<String, LocationSummary>?
Why not just using HashMap<String, String>() or a SimpleObject().. you can get rid of the LocationSummary

Reply:
updated with a hashmap